creator-node/src/components/tracks/TrackTranscodeHandoffManager.js

```diff
@@ -337,36 +340,42 @@ class TranscodeDelegationManager {
     fileNameNoExtension
   }) {
     const originalTrackFormData =
-      await TranscodeDelegationManager.createFormData(fileDir + '/' + fileName)
+      await TrackTranscodeHandoffManager.createFormData(
+        fileDir + '/' + fileName
+      )
 
     // TODO: make this constant. perhaps in a class
     const spID = config.get('spID')
     const { timestamp, signature } =
       generateTimestampAndSignatureForSPVerification(spID, DELEGATE_PRIVATE_KEY)
 
     const resp = await Utils.asyncRetry({
+      logger: TrackTranscodeHandoffManager.logger,
       asyncFn: axios,
-      asyncFnParams: {
-        url: `${sp}/transcode_and_segment`,
-        method: 'post',
-        data: originalTrackFormData,
-        headers: {
-          ...originalTrackFormData.getHeaders()
-        },
-        params: {
-          uuid: fileNameNoExtension,
-          timestamp,
-          signature,
-          spID
-        },
-        adapter: require('axios/lib/adapters/http'),
-        // Set content length headers (only applicable in server/node environments).
-        // See: https://github.com/axios/axios/issues/1362
-        maxContentLength: Infinity,
-        maxBodyLength: Infinity,
-        timeout: SEND_TRANSCODE_AND_SEGMENT_REQUEST_TIMEOUT_MS
-      },
-      asyncFnTask: 'transcode and segment'
+      asyncFnParams: [
+        {
+          url: `${sp}/transcode_and_segment`,
+          method: 'post',
+          data: originalTrackFormData,
+          headers: {
+            ...originalTrackFormData.getHeaders()
+          },
+          params: {
+            uuid: fileNameNoExtension,
+            timestamp,
+            signature,
+            spID
+          },
+          adapter: require('axios/lib/adapters/http'),
+          // Set content length headers (only applicable in server/node environments).
+          // See: https://github.com/axios/axios/issues/1362
+          maxContentLength: Infinity,
+          maxBodyLength: Infinity,
```

Comment:
interesting, just seeing this code for first time as well. seems dangerous to set to Infinity, but i can see the GH issue (thanks for linking to that here)
we should maybe just set this as the axios default somewhere upstream, but that's separate work

Reply:
I don't think it's your first time 🌚  hehehe
https://github.com/AudiusProject/audius-protocol/blame/0bfe7da35fb724b3869263f433230de74f923041/libs/src/services/creatorNode/CreatorNode.ts#L954L-L960

Reply:
lollll, clearly not
i guess i forgot - glad we're tracking the GH issue here tho
added a card to move this to global axios default - https://linear.app/audius/issue/CON-123/set-cn-axios-default-maxcontentlength-and-maxbodylength-to-infinity